source/org/zfin/db/postGmakePostloaddb/1136/ZFIN-8169.sql

```diff
@@ -0,0 +1,13 @@
+--liquibase formatted sql
+--changeset cmpich:ZFIN-8169
+
+create temp table tmp_id as
+select get_id('EXTNOTE')
+from single;
```

Comment:
Is "from single" necessary? Not a big deal, just caught my eye.

Reply:
Correct. It is not needed. That was the syntax with Informix.